dcat/rdf/dcat3.ttl

```diff
@@ -1,132 +1,152 @@
+      a foaf:Person ;
+      foaf:name "Simon J D Cox" ;
+      foaf:workInfoHomepage <http://people.csiro.au/Simon-Cox> ;
+      org:memberOf [
+          foaf:homepage <https://www.csiro.au> ;
+          foaf:name "Commonwealth Scientific and Industrial Research Organisation" ;
         ] ;
-      foaf:name "Rufus Pollock" ;
+      rdfs:seeAlso <https://orcid.org/0000-0002-3884-3420> ;
     ] ;
-  dcterms:contributor [
-      sdo:affiliation [
+  bibo:editor [
+      a foaf:Person ;
+      foaf:homepage <https://agbeltran.github.io> ;
+      foaf:name "Alejandra Gonzalez-Beltran" ;
+      org:memberOf [
           foaf:homepage <http://stfc.ac.uk> ;
           foaf:name "Science and Technology Facilities Council, UK" ;
         ] ;
       rdfs:seeAlso <https://orcid.org/0000-0003-3499-8262> ;
-      foaf:homepage <https://agbeltran.github.io> ;
-      foaf:name "Alejandra Gonzalez-Beltran" ;
     ] ;
-  dcterms:contributor [
-      sdo:affiliation [
-          foaf:homepage <http://www.refinitiv.com> ;
-          foaf:name "Refinitiv" ;
-        ] ;
-      foaf:name "David Browning" ;
-    ] ;
-  dcterms:contributor [
-      rdfs:seeAlso <http://makxdekkers.com/makxdekkers.rdf#me> ;
-      foaf:homepage <http://makxdekkers.com/> ;
-      foaf:name "Makx Dekkers" ;
-    ] ;
-  dcterms:contributor [
-      rdfs:seeAlso <http://www.eurecom.fr/~atemezin/gatemezing-foaf.rdf> ;
-      foaf:name "Ghislain Auguste Atemezing" ;
-    ] ;
-  dcterms:contributor [
-      rdfs:seeAlso <https://jakub.klímek.com/#me> ;
-      foaf:homepage <https://jakub.klímek.com/> ;
-      foaf:name "Jakub Klímek" ;
-    ] ;
-  dcterms:contributor [
-      rdfs:seeAlso <https://orcid.org/0000-0001-5648-2713> ;
-      foaf:homepage <http://www.imati.cnr.it/index.php/people/8-curricula/178-riccardo-albertoni> ;
-      foaf:homepage <https://w3id.org/people/ralbertoni/> ;
-      foaf:name "Riccardo Albertoni" ;
-    ] ;
-  dcterms:contributor [
-      rdfs:seeAlso <https://orcid.org/0000-0001-9300-2694> ;
-      foaf:homepage <https://www.andrea-perego.name/foaf/#me> ;
+  bibo:editor [
+      a foaf:Person ;
       foaf:name "Andrea Perego" ;
+      rdfs:seeAlso <https://orcid.org/0000-0001-9300-2694> ;
     ] ;
-  dcterms:contributor [
+  bibo:editor [
+      a foaf:Person ;
+      foaf:name "Peter Winstanley" ;
+    ] ;
+  bibo:translator [
+      a foaf:Person ;
       foaf:homepage <http://www.asahi-net.or.jp/~ax2s-kmtn/> ;
       foaf:name "Shuji Kamitsuna" ;
     ] ;
-  dcterms:contributor [
-      foaf:name "Boris Villazón-Terrazas" ;
+  bibo:translator [
+      a foaf:Person ;
+      foaf:homepage <https://jakub.klímek.com/> ;
+      foaf:name "Jakub Klímek" ;
+      rdfs:seeAlso <https://jakub.klímek.com/#me> ;
     ] ;
-  dcterms:contributor [
-      foaf:name "Marios Meimaris" ;
+  bibo:translator [
+      a foaf:Person ;
+      foaf:name "Fadi Maali" ;
+      org:memberOf [
+          foaf:homepage <http://www.deri.ie/> ;
+          foaf:name "DERI, NUI Galway" ;
```

Comment:
DERI no longer exists. 
Some of its work continues in the Data Science Institute. https://dsi.nuigalway.ie/

Reply:
Thanks, @dr-shorthair.
http://www.deri.ie/ resolves to  https://dsi.nuigalway.ie/ .  
According to his LinkedIn profile,  Fadi Maali has never worked at dsi.nuigalway.ie, though he worked at DERI at the time of his contribution. 

Replacing http://www.deri.ie/ with https://dsi.nuigalway.ie/  seems quite imprecise. 

We might consider deleting the link, as it can break in the future, but even a broken URI brings some value when considered jointly with other crawled info. 

I would be tempted to leave things as they are. What do you think?

Reply:
Yes, OK.